packages/aws-cdk-lib/aws-eks/test/cluster.test.ts

```diff
@@ -27,7 +27,7 @@ describe('cluster', () => {
     const cluster = new eks.Cluster(stack, 'Cluster', {
       version: CLUSTER_VERSION,
       albController: {
-        version: eks.AlbControllerVersion.V2_4_1,
+        version: eks.AlbControllerVersion.V2_6_2,
```

Comment:
I would suggest leave this untouched. I know V2_4_1 is not the latest version but I would prefer bump this version when we really have to.

Reply:
sure thing, I will revert this change.